vuepress/docs/next/docs/releases/README.md

```diff
@@ -1,10 +1,197 @@
+* The latest MySql 8.x and PostgreSQL 14 versions are supported.
+* Oracle 18c as an external DBMS is supported.
+* Infinispan is now used only for EAP session sharing. Object cache is implemented in Redis.
+* Logging infrastructure has been moved to Logback.
+* CMS content API performance optimizations have been implemented.
+
+### App Builder
+
+* Added dash (-) support for fragment code validation.
+* Improved UI performance affected by instances containing significant content and assets.
+* Implemented performance improvements in page management.
+* Resolved several extended bugs hindering performance and user experience.
+* Reduced the size of the application state and started phasing out obsolete technologies.
+
+### Entando Component Generator (ECG)
+
+* Upgraded JHipster to version 7.2.
+* ECG now allows the use of a BOM curated by Entando and containing more recent dependencies than is standard for a specific version of JHipster.
+* Removed the .map files generated from Blueprint-generated code.
+* Implemented bug fixes and error handling improvement.
+* Switched to bundle plugin descriptor version 3.
+
+### Entando Component Repository (ECR)
+
+* Added support for bundle upgrade, downgrade and update.
+* Bundle improvements; in particular, bundle plugin security and reliability via signing the names.
+* It's now possible to specify standard and secret environment variables for a bundle plugin (microservice).
+* Offering experimental support for bundle plugin configuration profiles.
+* Offering better support for bundles and bundle plugins with very long names.
+* Implemented better management of certain error conditions during installation and updates.
+* Introduced system-level bundles, which allow the installation of global static resources.
+* The ECR client now explicitly marks bundles that are installed with an amended installation plan and allows the plan to be recalled.
+* Added support for Oracle 18c.
+* Fixed a bug that caused the ECR to indefinitely stay in busy-state due to particular failure conditions.
+* Fixed a bug that (under certain conditions) prevented the bundle plugin pod from properly restarting when scaled down, then back up again.
+* The ECR now correctly deletes the plugin pod when the bundle is uninstalled, but the deployment and volumes are retained.
+
+### ENT (entando-cli)
+
+* Reworked the `ent diag` command to now also export CRs, ingresses, deployments, pod "previous" logs, namespace events, etc.
+* ent is now able to sync the kubectl version of the current profile with the server version (`ent auto-align-kubectl`)
+* Added helper to fetch logs from bundle plugins (`ent prj be-log`).
+* In order to run node-based Entando tools, ent now installs and uses a private copy of node at user-level.
+* A jq internal dependency is now installed at user-level, privately to ent.
+* Added the command `ent pkg` (to manage packages installed at user-level) with initial support for jq and K9s.
```

Comment:
@Lyd1aCla1r3 Could you please review this list and add notes to the CLI ticket to make sure we include/update anything referenced here?  Then we can link to the appropriate section if you think it makes sense.

Reply:
@nshaw done